docs/src/main/asciidoc/elasticsearch.adoc

```diff
@@ -6,14 +6,13 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 = Connecting to an Elasticsearch cluster
 include::_attributes.adoc[]
 :categories: data
-:summary: This guide covers how to use an Elasticsearch cluster using the low level or high level REST clients.
+:summary: This guide covers how to use an Elasticsearch cluster using the low level REST client or the Java API client.
 
 Elasticsearch is a well known full text search engine and NoSQL datastore.
 
 In this guide, we will see how you can get your REST services to use an Elasticsearch cluster.
 
-Quarkus provides two ways of accessing Elasticsearch: via the lower level `RestClient` or via the `RestHighLevelClient` we will call them
-the low level and the high level clients.
+Quarkus provides three ways of accessing Elasticsearch: via the lower level `RestClient`, via the high level `RestHighLevelClient`, or via the Java API client.
```

Comment:
Unrelated to your PR, but shouldn't we just drop the high-level client?

It was deprecated in Quarkus 2, though of course the value of that deprecation is debatable since there wasn't any alternative...

@gsmet any opinion?

Reply:
(forgot to post my review)

I agree with this, I will draft a proposal in a separate commit.